lib/widgets/message_list.dart

```diff
@@ -395,9 +396,14 @@ class MarkAsReadWidget extends StatelessWidget {
             padding: const EdgeInsets.all(10),
             child: FilledButton.icon(
               style: FilledButton.styleFrom(
-                padding: const EdgeInsets.all(10),
-                textStyle: const TextStyle(fontSize: 18, fontWeight: FontWeight.w200),
-                shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(5)),
+                backgroundColor: const HSLColor.fromAHSL(1,227,0.78,0.59).toColor(),
```

Comment:
nit: space after comma

Also Chris observed in that chat thread that this is the same color as the unread markers on messages, and that seems like not a coincidence — those markers are what this button will eliminate, and the shared color draws the connection. So let's factor it out with a name. Can be a static on `_UnreadMarker`, like `static final color = …`.